### ISSUE TITLE
please give dates for usage data

### ISSUE THREAD

**Issue by chaals** [schemaorg/schemaorg#2011]
The usage data for terms is vague, and presumably comes from some partial crawl of mostly-public data so it should be, but it would help to know whether it is from several years ago (which is my actual assumption) or last week - and in an ideal world to see uptake by colleccting it every few years.

**Comment by unor** [comment]
→ #489

**Comment by chaals** [comment]
@unor yes - but it would independently be helpful to have the date where the numbers are, i.e. on each page that has them.
